# Datos del alumno:

Doble click para responder
>
>**Nombre y apellido: Karla Blanco**
>
>

# Pregunta 10 Prueba 2

Las importaciones de bienes en Chile pueden estar influenciadas por diversas variables económicas y comerciales. En nuestro país, el nivel de importaciones (Y) está relacionado con las exportaciones (X) de nuestros bienes. Para estudiar esta relación, usted dispone de una base de datos que contiene las exportaciones e importaciones de Chile en millones de dólares desde enero de 2013 hasta noviembre de 2023

Descargue la base de datos **<a href="https://docs.google.com/spreadsheets/d/1mUXMrL59Uy8anOFAAFFw4lGSmYm1ABjW/export?format=xlsx">aquí</a>** y responda las siguientes preguntas.

In [ ]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
from sklearn.metrics import r2_score
from scipy.stats import norm

df = pd.read_excel('Base_P10_3.xlsx')
df.head()

Descripción series  Exportaciones (millones de dólares)  \
0         2013-01-01                          6847.655714   
1         2013-02-01                          5606.281024   
2         2013-03-01                          6206.699426   
3         2013-04-01                          6803.490512   
4         2013-05-01                          7036.902907   

   Importaciones (millones de dólares)  
0                          7032.761626  
1                          6138.969145  
2                          6157.193733  
3                          6795.003927  
4                          7071.360633

a) El ministro de Economía señaló en una presentación que, a medida que aumentan las exportaciones, también aumentan las importaciones. Determine el modelo de regresión lineal correspondiente a los datos y utilícelo para confirmar o refutar la afirmación del ministro. **(3 puntos)**

In [ ]:
v_indep = df['Exportaciones (millones de dólares)'].copy()
v_depen = df['Importaciones (millones de dólares)'].copy()

pendiente, intercepto = np.polyfit(v_indep,v_depen,1)

print(f"La pendiente es {pendiente:.2f} y el intercepto es {intercepto:.2f}")

La pendiente es 0.90 y el intercepto es 457.59


**Respuesta:**
Se puede visualizar que la pendiente es positiva.
Es decir, que cuando las exportaciones aumentan, las importaciones también aumentarán. Por lo que, la relación es directa.



b) Calcule e interprete el coeficiente de determinación. **(1 punto)**

In [ ]:
#Definición de variables independiente y dependiente
v_indep = df['Exportaciones (millones de dólares)'].copy()
v_depen = df['Importaciones (millones de dólares)'].copy()

#Definición de función lineal encontrada con modelo de regresión
def f(x):
    return 0.90*x+457.59

vd_prediccion=f(v_indep) #donde f es la función encontrada en el modelo de regresión lineal

coef_determinacion = r2_score(v_depen, vd_prediccion)
round(coef_determinacion,2)

0.68

**Interpretación**
El coeficiente de determinación es aproximadamente 0.68, lo que significa que el 68% de las variaciones en las importaciones pueden ser explicadas por las exportaciones. Por lo tanto, el modelo tiene un nivel de confialibilidad moderado-alto.

c) En diciembre de 2023 las exportaciones fueron de 8.100 millones de dólares. Según el modelo lineal, ¿cuál es el monto estimado de las importaciones de ese mes? **(2 puntos)**

In [ ]:
f(8100)



7747.59

**Respuesta**
El monto estimado de las importaciones es aprox 7747.59 millones

💾 **Recuerda:** Una vez que hayas respondido todas las preguntas, descarga este archivo para subirlo a AVA.
>
> Para hacerlo, ve al menú superior y selecciona:  
> **Archivo → Descargar → Descargar .ipynb**